Xamarin.Forms.Core.UnitTests/ImageTests.cs

```diff
@@ -141,6 +141,62 @@ public void TestSource ()
 			Assert.True (signaled);
 		}
 
+		[Test]
+		public void TestErrorSource()
+		{
+			var image = new Image
+			{
+				Source = "File.png",
+				ErrorSource = null
+			};
+
+			Assert.IsNotNull(image.Source);
+			Assert.IsNull(image.ErrorSource);
+			bool signaled = false;
+			
+			image.PropertyChanged += (_, e) =>
+			{
+				if (e.PropertyName == Image.ErrorSourceProperty.PropertyName)
+					signaled = true;
+			};
+
+			var errorSource = ImageSource.FromFile("error.png");
+
+			image.ErrorSource = errorSource;
+
+			Assert.IsNotNull(image.ErrorSource);
+			Assert.AreEqual(image.ErrorSource, errorSource);
+			Assert.IsTrue(signaled);
+		}
+
+		[Test]
+		public void TestLoadingSource()
+		{
+			var image = new Image
+			{
+				Source = "File.png",
+				LoadingSource = null
+			};
+
+			Assert.IsNotNull(image.Source);
+			Assert.IsNull(image.LoadingSource);
+			bool signaled = false;
+
+			image.PropertyChanged += (_, e) =>
+			{
+				if (e.PropertyName == Image.LoadingSourceProperty.PropertyName)
+					signaled = true;
+			};
+
+			var errorSource = ImageSource.FromFile("error.png");
```

Comment:
Change this name to loadingSource

Also can you have this same type of test on the other elements? (Button, ImageButton, etc...)

I fixed a copy/paste BindableProperty error on Button that I think would have been picked up by a test like this on Button